### PR TITLE
chore: use ecr registry and hardcode service account

### PR DESCRIPTION
**Pull request by yashmehrotra**

### CODE REVIEW

File: chart/templates/postgres-statefulset.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
       - name: postgresql
-        image: postgres:14
+        image: public.ecr.aws/k4y9r6y5/postgres:14.1.0.21
```
Comment (moshloop) [inline chart/templates/postgres-statefulset.yaml:23]: Shouldn't this be global registry ?

File: chart/values.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@
 # Declare variables to be passed into your templates.
 
 global:
+  imageRegistry: public.ecr.aws/k4y9r6y5
```
Comment (moshloop) [inline chart/values.yaml:6]: lets keep the default in the charts to docker and set it to ECR only in the mission-control chart

File: .github/workflows/helm-test.yml [hunk 3]
```diff
@@ -47,6 +47,17 @@ jobs:
       - name: Install helm chart
         run: 'helm install canary-checker canary-checker-1.0.0.tgz -n canary-checker --create-namespace'
 
+      - name: Debug
```
Comment (moshloop) [inline .github/workflows/helm-test.yml:50]: Is this still required

Comment (yashmehrotra) [reply .github/workflows/helm-test.yml]: My bad, removed it

**Pull request merged**